### PR TITLE
Exam

### PR DESCRIPTION
**Pull request by TamirDwek**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+img{
+    width: 150px;
+    height: 150px;
```
Comment (shaharsol) [inline assets/css/style.css:3]: to scale down images you should pick either width *or* height but not both. if the original image is 400x200, this scale will cause distortion

File: assets/js/app.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+function generateHTML(data){
+const newHTML = `
+<tr>
+<td> ${data.productName} </td>
+<td> ${data.productPrice} </td>
+<td> ${data.productCategory} </td>
+<td><img src="${data.imageUrl}"></td>
+</tr>
```
Comment (shaharsol) [inline assets/js/app.js:22]: indentation

File: assets/js/app.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+
+function renderHTML(newHTML) {
+const productContainer = document.getElementById('productContainer')
+productContainer.innerHTML += newHTML
+}
+
+function clearForm()
+{const registrationForm = document.getElementById('registrationForm')
+    registrationForm.reset()
```
Comment (shaharsol) [inline assets/js/app.js:35]: structure. use option+shift+f to fix the code structure automatically

File: assets/js/app.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+// //     if(carsJSON) {
+// //         const cars = JSON.parse(carsJSON)
+// //         for(const car of cars) {
+// //             const newHTML = generateHTML(car)
+// //             renderHTML(newHTML)
+// //         }
+// //     }
+// // }
+
+// // loadCarsFromLocalStorage()
```
Comment (shaharsol) [inline assets/js/app.js:114]: please don't leave commented out code in PRs

**Comment by shaharsol** [comment]
works beautifully. perfect. שכוייח

**Comment by TamirDwek** [comment]
I will follow the comments.
Thank you Shahar

**Pull request closed without merging**